pkg/server/router.go

```diff
@@ -53,6 +56,14 @@ func (r *Router) Routes(h router.Handlers) http.Handler {
 	m.PathPrefix("/dashboard/").Handler(vueUI.IndexFileOnNotFound())
 	m.PathPrefix("/api-ui").Handler(vueUI.ServeAsset())
 
+	sbDownloadHandler := supportbundle.NewDownloadHandler(r.server.Context, r.server.namespace, r.scaled)
```

Comment:
The namespace is available from r.options.namespace. We probably don't need to add namespace to server structure

Reply:
fixed! thanks for the tip. https://github.com/harvester/harvester/pull/689/commits/054c7f9499d73ea94a99588955fc9c405573c26b